chapters/ch16.md

```diff
@@ -1,70 +1,71 @@
 # 16. Cleaning up mislabeled dev and test set examples
 ->
-
+# 16. Dọn dẹp những mẫu bị gán nhãn nhầm trong tập phát triển và tập kiểm tra
 
 During error analysis, you might notice that some examples in your dev set are mislabeled. When I say “mislabeled” here, I mean that the pictures were already mislabeled by a human labeler even before the algorithm encountered it. I.e., the class label in an example ​(x,y)​ has an incorrect value for ​y.​ For example, perhaps some pictures that are not cats are mislabeled as containing a cat, and vice versa. If you suspect the fraction of mislabeled images is significant, add a category to keep track of the fraction of examples mislabeled:
 ->
+Trong quá trình phân tích lỗi, bạn có thể nhận thấy rằng một vài mẫu trong trong tập phát triển đã bị gán nhãn nhầm. Khi tôi nói "bị gán nhãn nhầm" ở đây, ý tôi là những tấm ảnh đã bị gán nhãn nhầm bởi người dán nhãn trước cả khi thuật toán thuật toán gặp nó. Hay nói cách khác, nhãn lớp của một mẫu (x,y) có một giá trị sai cho y.  Ví dụ, có thể một số ảnh không chứa mèo bị gán nhãn nhầm thành có mèo, và ngược lại. Nếu bạn nghi ngờ rằng tỷ lệ những ảnh bị gán nhãn nhầm là quan trọng, hãy thêm một hạng mục để theo dõi tỷ lệ các mẫu bị gán nhãn nhầm:
 
 ![img](../imgs/C16_01.png)
 
 
 Should you correct the labels in your dev set? Remember that the goal of the dev set is to help you quickly evaluate algorithms so that you can tell if Algorithm A or B is better. If the fraction of the dev set that is mislabeled impedes your ability to make these judgments, then it is worth spending time to fix the mislabeled dev set labels.
 ->
-
+Bạn có nên sửa lại những nhãn trong tập phát triển không? Hãy nhớ rằng mục tiêu của tập phát triển là giúp bạn nhanh chóng đánh giá những thuật toán nhờ đó bạn có thể biết Thuật toán A hay Thuật toán B là tốt hơn. Nếu tỷ lệ bị gán nhãn nhầm trong tập phát triển cản trở khả năng ra những quyết định này của bạn, thì sẽ là đáng bỏ thời gian ra để sửa lại những nhãn bị gán nhầm của tập phát triển.
```

Comment:
> nên sửa lại những nhãn trong tập phát triển
mình nghĩ nên là "nên sửa lại những nhãn "sai" trong tập phát triển"?

Reply:
Vì tác giả đã nói về những nhãn sai trong tập phát triển ở đoạn trên nên người đọc có thể hiểu ngầm là sửa lại những nhãn sai. Theo mình, không cần phải thêm từ "sai" ở đây để không bị lặp quá nhiều. Mình tin đó cũng là chủ ý của tác giả.

Reply:
Mình nghĩ không thêm từ "sai" cũng, giữ nguyên cách dịch cũ cũng ok. Tác giả thì chắc không có chủ ý gì, vì correct the labels = modify the wrong labels.

Reply:
@rootonchair theo mình để "sửa những nhãn sai" sẽ bớt gây nhầm lẫn. Tác giả cũng dùng từ "correct" (động từ) chứ không dùng từ "change".

Reply:
@william-vu , @tiepvupsu mình/em đã sửa lại thành "những nhãn sai"